app/views/admin/organizations/index.html.haml

```diff
@@ -1,33 +1,38 @@
-%header.with-subtitle
-  %h1
-    Manage Organizations
-  = link_to 'New Organization', new_admin_organization_url, class: "button-green sharing-links"
+.row
+  .col-md-9
+    %h1
+      Manage Organizations
+  .col-md-3
+    = link_to 'New Organization', new_admin_organization_url, class: "btn btn-success float-right"
 
 %h3
   #{number_with_delimiter(@organizations_count)} organizations
   = (params[:query].present? || params[:is_paid].present?) ? 'matching' : 'in the system'
 
-%div{ style: "width: 100%; text-align: right;" }
   = form_tag admin_organizations_path, method: :get do
-    = hidden_field_tag :sort, @sort
-    = hidden_field_tag :sort_direction, @sort_direction
-    %label{ style: 'display: inline-block; margin-right: 1em;' }
-      = check_box_tag :is_paid, true, params[:is_paid]
-      paid only
-    = text_field_tag :query, params[:query], placeholder: "Find name, location #{current_user.developer? ? 'or paid features' : '' }", style: "margin: 0; width: 30%; min-width: 200px;"
-    = submit_tag 'Search', name: 'search', class: 'button-blue'
+    .row
+      .col-md-6
+        = hidden_field_tag :sort, @sort
+        = hidden_field_tag :sort_direction, @sort_direction
+      .col-md-2
+        %label.mt-4
+          = check_box_tag :is_paid, true, params[:is_paid]
+          Paid only
+      .col-md-3
+        = text_field_tag :query, params[:query], placeholder: "Find name, location #{current_user.developer? ? 'or paid features' : '' }", class: "form-control mt-4"
+      .col-md-1
+        = submit_tag 'Search', name: 'search', class: 'btn btn-primary mt-4'
 
 - if @organizations.present?
-  .paginate-container
-    = paginate @organizations
+  = paginate @organizations, views_prefixs: "admin"
 
-%section.full-screen-table
+.full-screen-table-wrapper
   %table.table.table-striped.table-bordered
-    %thead
+    %thead.thead-light
       - ['name', 'approved', 'created_at'].each do |col|
         - active_col = @sort == col
         %th
-          %a.column_sort_update{ href: '#', class: ('active' if active_col), style: 'display: block;', data: { sort: col, direction: (active_col && @sort_direction == 'desc' ? 'asc' : 'desc') } }
```

Comment:
This is its own legacy sortable functionality (check in `app/views/admin/organizations/index.html.haml`)

We have a new sortable functionality! Using a concern and some helpers. It's viewable in the paid_features controller index. This should be switched over to that!